mesos-docker/run/run.sh

```diff
@@ -190,9 +196,17 @@ function start_master {
 
   dip=$(docker_ip)
 
-  start_master_command="/usr/sbin/mesos-master --ip=$dip $(quote_if_non_empty $MESOS_MASTER_CONFIG)"
+  if [[ -n $INSTALL_ZK ]]; then
+    zk="--zk=zk://$dip:2181/mesos"
+  fi
+
+  start_master_command="/usr/sbin/mesos-master $zk --ip=$dip $(quote_if_non_empty $MESOS_MASTER_CONFIG)"
   start_master_command="$(get_mesos_update_command) ; $start_master_command"
 
+
+  #upmesos="apt-get update -o Dir::Etc::sourcelist=sources.list.d/mesosphere.list -o Dir::Etc::sourceparts=- -o APT::Get::List-Cleanup=0"
+  #start_master_command="$upmesos ; apt-get install --only-upgrade mesos; $start_master_command"
```

Comment:
Are these comments still needed here? Generally we don't commit commented out code, since there's Git for history.